github/graphql/client.go

```diff
@@ -87,12 +118,20 @@ func newClientWithTransport(tr http.RoundTripper) Client {
 // It expects the client to perform any authentication that is necessary.
 func NewClientWithHTTP(http *http.Client) Client {
 	return &client{
-		http: http,
+		http:  http,
+		pkeys: make(pkeys),
 	}
 }
 
+type pkeys map[int64]*rsa.PrivateKey
+
 type client struct {
-	http *http.Client
+	mu    sync.Mutex
```

Comment:
```suggestion
	mu    sync.RWMutex
```

Also, follow the ["mutex hat"](https://dmitri.shuralyov.com/idiomatic-go#mutex-hat) convention.

Reply:
Done.